src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Model/PowerShellInstalledPackage.cs

```diff
@@ -43,5 +87,27 @@ internal static List<PowerShellInstalledPackage> GetPowerShellPackageView(Dictio
 
             return views;
         }
+
+        private static PackageReaderBase GetPackageReader(string installPath, PackageIdentity package)
+        {
+            FileInfo nupkg = null;
+            if (Directory.Exists(installPath))
```

Comment:
The directory check isn't needed FileInfo can handle missing directories in the .Exists check I believe.